lms/djangoapps/completion/migrations/0001_initial.py

```diff
@@ -7,6 +7,7 @@
 from django.conf import settings
 import model_utils.fields
 import openedx.core.djangoapps.xmodule_django.models
+import openedx.core.djangolib.fields
```

Comment:
Should we make this import in the migration the same as the one in `models.py`? Otherwise this will presumably generate a spurious and invalid migration when we upgrade django since the model will no longer match the migration history.

```python
try:
    from django.models import BigAutoField  # New in django 1.10
except ImportError:
    from openedx.core.djangolib.fields import BigAutoField
```